packages/react-vapor/tsconfig.json

```diff
@@ -10,7 +10,8 @@
         "removeComments": true,
         "noUnusedLocals": true,
         "lib": ["es2015", "dom"],
-        "allowSyntheticDefaultImports": true

```

Comment:
How do you know that this won't introduce breaking changes? Or is the fact that because you've introduced breaking changes in the slider anyway means you're more comfortable doing this?

Reply:
I didn't remove the line, only added a comma at the end of the line. GitHub shows the diff in a weird way :stuck_out_tongue:

Reply:
I meant the next line, sorry. Put the comment in the wrong place

Reply:
Oh :see_no_evil: `strictBindCallApply` is only for a stricker type checking inside the package. It shouldn't have any impact on the distributed code or types.
![image](https://user-images.githubusercontent.com/35579930/85073484-dc6cae00-b188-11ea-9ad9-037fab896e25.png)
https://www.typescriptlang.org/docs/handbook/compiler-options.html